backend/src/auth/strategy/jwt.strategy.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { JwtService } from '@nestjs/jwt';
+import { PrismaService } from '../../prisma/prisma.service';
```

Comment:
use path aliases
@/modules/prisma/prisma.service or something like that, it's will help if you trying to refactor code